tools/test.py

```diff
@@ -174,6 +174,16 @@ def main():
                               **cfg.data.get('test_dataloader', {}))
     data_loader = build_dataloader(dataset, **dataloader_setting)
 
+    if cfg.model.get('backbone', None):
+        # remove pretrain steps for testing
+        cfg.model.backbone.pretrained = None
+        cfg.model.backbone.pretrained2d = False
```

Comment:
maybe some `model.backbone` has no pretrained2d. such as C3D?

Reply:
It doesn't matter, right?

Reply:
Oops